challenges/objects-arrays.js

```diff
@@ -6,29 +6,68 @@
   object name, diet, weight, length, period
 */
 
+function Dinosaur (name, diet, weight, length, period){
+  this.name = name;
+  this.diet = diet;
+  this.weight = weight;
+  this.length = length;
+  this.period = period;
+}
+
+const joe = new Dinosaur ('tyrannosaurus', 'carnivorous', '7000kg', '12m', 'Late Cretaceous');
+
+const bob = new Dinosaur ('stegosaurus', 'herbivorous', '2000kg', '9m', 'Late Jurrasic');
+
+const tom = new Dinosaur ('velociraptor', 'carnivorous', '15kg', '1.8m', 'Late Cretaceous');
+
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceous
 
+// const joe = {
+//   name: 'tyrannosaurus',
+//   diet: 'carnivorous',
+//   weight: 7000 + 'kg',
```

Comment:
This is really all that was asked, but again I love that you went the other route and got those extra reps in.